dali/python/nvidia/dali/ops.py

```diff
@@ -613,19 +545,112 @@ def _load_ops():
     global _cpu_ops
     global _gpu_ops
     global _mixed_ops
-    _cpu_ops = _cpu_ops.union(set(b.RegisteredCPUOps()))
-    _gpu_ops = _gpu_ops.union(set(b.RegisteredGPUOps()))
-    _mixed_ops = _mixed_ops.union(set(b.RegisteredMixedOps()))
+    _cpu_ops = _cpu_ops.union(set(_b.RegisteredCPUOps()))
+    _gpu_ops = _gpu_ops.union(set(_b.RegisteredGPUOps()))
+    _cpu_ops.remove("ExternalSource")
```

Comment:
It should not be called like that

Reply:
I want to have "ExternalSource" (not "_ExternalSource"!) in C++ - and RegisteredGPUOps just gets all ops from schema registry. How do you propose to solve this?